core/sql/cli/Context.cpp

```diff
@@ -3850,6 +3856,48 @@ void ContextCli::createMxcmpSession()
   mxcmpSessionInUse_ = TRUE;
 }
 
+// ----------------------------------------------------------------------------
+// Method:  updateMxcmpSession
+//
+// Updates security attributes in child arkcmp
+//
+// Returns: 0 = succeeded; -1 = failed
+// ----------------------------------------------------------------------------
+Int32 ContextCli::updateMxcmpSession()
+{
+  // If no child arkcmp, just return
+  if (getArkcmp()->getServer() == NULL)
```

Comment:
At the time ContextCli is instantiated, the first entry in arkcmpArray_ is created.  So there should always be entry 0.  The method getArkcmp() returns the 0 entry if none is specified.  Therefore, getArkcmp() should not be NULL.